extensions/github/src/pushErrorHandler.ts

```diff
@@ -7,7 +7,7 @@ import { TextDecoder } from 'util';
 import { commands, env, ProgressLocation, Uri, window, workspace, QuickPickOptions, FileType, l10n } from 'vscode';
 import { getOctokit } from './auth';
 import { GitErrorCodes, PushErrorHandler, Remote, Repository } from './typings/git';
-import path = require('path');
```

Comment:
FYI this is what the typescript extension spit out when I typed `path` in my editor.

Reply:
Hm, yeah that seems wrong but it may be because of how the tsconfig is setup. Will ask the ts team about it

Should be prevented entirely though once I updated the tsconfig to use the new `bundler` option for `moduleResolution`